src/borg/archiver.py

```diff
@@ -2422,6 +2422,17 @@ def main():  # pragma: no cover
         tb_log_level = logging.ERROR if important else logging.DEBUG
         tb = sysinfo()
         exit_code = EXIT_ERROR
+    except OSError as e:
+        at_frame = traceback.extract_tb(e.__traceback__, limit=-1)[0]
+        msg = textwrap.dedent("""
+        I/O error: {error}
+
+        At: {frame.filename}:{frame.lineno} ({frame.name})
```

Comment:
if the code where it crashes is some low-level routine that is called from a lot of places, this is almost as useless as attic's "AttributeError" (it did not say what/where) message.

as people usually do not run on debug level, we will get bug reports with not enough information.
and if the issue is not easily reproducable, that will be all we have.